### PR TITLE
Admin Page: readme.md - Add reference to development and production builds

### LINKED ISSUE #4403: Add reference to production build in the README.md
We now have `npm run build` for creating a development build and `npm run build-production` for creating optimized builds (deduped and minified/uglified)

### PR DESCRIPTION
**Pull request by oskosk**
Fixes #4403 .

#### Changes proposed in this Pull Request:

- Adds 2 new sections to the readme: Development build, Production build and Development build with changes monitoring.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -70,8 +70,26 @@ Known Issues:
...
+
 #### Development build
 
+The development build will create a build without minifying or deduping code.
+
+```
+npm run build
+```
+
+This will set an environment variable `NODE_ENV=production` before doing a `npm run build`.
```
Comment (zinigor) [inline readme.md]: I believe that is supposed to be in the production build section, right? Also, the production build will prepare language files, which will take some time.

Comment (oskosk) [inline readme.md]: Right! I'm updating this branch

**Comment by jeherve** [comment]
🚢

**Comment by oskosk** [comment]
niceee.

**Pull request merged**